### ISSUE TITLE
Level3 asn attempted for los/jitter filesets, which stop at Level1b

### ISSUE THREAD

**Issue by stscijgbot-jp** [spacetelescope/jwst#7214]
_Issue [JP-2703](https://jira.stsci.edu/browse/JP-2703) was created on JIRA by [Hien Tran](https://jira.stsci.edu/secure/ViewProfile.jspa?name=htran):_

The science members for commissioning visit 1439:1:1 consist of exposures only of the los/jitter type (DMS_NOTE = WFSC_LOS_JITTER), which ends at level 1b. Despite this, l3 asn is attempted, causing it to be stuck forever in data collection
```
asn table file: /ifs/archive/ops/jwst/info/owlmgr/paths/sdp/asn_polling/asn_table/jw01439-o001_20220723t113442_wfs-image3_001_asn.json
CHECKING PRODUCT: wfs-image3: jw01439-o001_t002_nircam_clear-f210m-sub8fp1a-nrca3_{suffix}-02
NOT READY: asn member /ifs/archive/ops/jwst/info/owlmgr/paths/sdp/asn_creation/cal/level3/jw01439001001_03102_00027_nrca3_cal.fits not yet available.
NOT READY: asn member /ifs/archive/ops/jwst/info/owlmgr/paths/sdp/asn_creation/cal/level3/jw01439001001_03102_00026_nrca3_cal.fits not yet available.
NOT READY: asn member /ifs/archive/ops/jwst/info/owlmgr/paths/sdp/asn_creation/cal/level3/jw01439001001_03102_00025_nrca3_cal.fits not yet available.
NOT READY: asn member /ifs/archive/ops/jwst/info/owlmgr/paths/sdp/asn_creation/cal/level3/jw01439001001_03102_00020_nrca3_cal.fits not yet available.
NOT READY: asn member /ifs/archive/ops/jwst/info/owlmgr/paths/sdp/asn_creation/cal/level3/jw01439001001_03102_00017_nrca3_cal.fits not yet available.
NOT READY: asn member /ifs/archive/ops/jwst/info/owlmgr/paths/sdp/asn_creation/cal/level3/jw01439001001_03102_00022_nrca3_cal.fits not yet available.
NOT READY: asn member /ifs/archive/ops/jwst/info/owlmgr/paths/sdp/asn_creation/cal/level3/jw01439001001_03102_00024_nrca3_cal.fits not yet available.
NOT READY: asn member /ifs/archive/ops/jwst/info/owlmgr/paths/sdp/asn_creation/cal/level3/jw01439001001_03102_00018_nrca3_cal.fits not yet available.
NOT READY: asn member /ifs/archive/ops/jwst/info/owlmgr/paths/sdp/asn_creation/cal/level3/jw01439001001_03102_00012_nrca3_cal.fits not yet available.
NOT READY: asn member /ifs/archive/ops/jwst/info/owlmgr/paths/sdp/asn_creation/cal/level3/jw01439001001_03102_00019_nrca3_cal.fits not yet available.
NOT READY: asn member /ifs/archive/ops/jwst/info/owlmgr/paths/sdp/asn_creation/cal/level3/jw01439001001_03102_00015_nrca3_cal.fits not yet available.
NOT READY: asn member /ifs/archive/ops/jwst/info/owlmgr/paths/sdp/asn_creation/cal/level3/jw01439001001_03102_00010_nrca3_cal.fits not yet available.
NOT READY: asn member /ifs/archive/ops/jwst/info/owlmgr/paths/sdp/asn_creation/cal/level3/jw01439001001_03102_00013_nrca3_cal.fits not yet available.
NOT READY: asn member /ifs/archive/ops/jwst/info/owlmgr/paths/sdp/asn_creation/cal/level3/jw01439001001_03102_00021_nrca3_cal.fits not yet available.
NOT READY: asn member /ifs/archive/ops/jwst/info/owlmgr/paths/sdp/asn_creation/cal/level3/jw01439001001_03102_00014_nrca3_cal.fits not yet available.
NOT READY: asn member /ifs/archive/ops/jwst/info/owlmgr/paths/sdp/asn_creation/cal/level3/jw01439001001_03102_00016_nrca3_cal.fits not yet available.
NOT READY: asn member /ifs/archive/ops/jwst/info/owlmgr/paths/sdp/asn_creation/cal/level3/jw01439001001_03102_00023_nrca3_cal.fits not yet available.
NOT READY: asn member /ifs/archive/ops/jwst/info/owlmgr/paths/sdp/asn_creation/cal/level3/jw01439001001_03102_00004_nrca3_cal.fits not yet available.
NOT READY: asn member /ifs/archive/ops/jwst/info/owlmgr/paths/sdp/asn_creation/cal/level3/jw01439001001_03102_00006_nrca3_cal.fits not yet available.
NOT READY: asn member /ifs/archive/ops/jwst/info/owlmgr/paths/sdp/asn_creation/cal/level3/jw01439001001_03102_00008_nrca3_cal.fits not yet available.
NOT READY: asn member /ifs/archive/ops/jwst/info/owlmgr/paths/sdp/asn_creation/cal/level3/jw01439001001_03102_00011_nrca3_cal.fits not yet available.
NOT READY: asn member /ifs/archive/ops/jwst/info/owlmgr/paths/sdp/asn_creation/cal/level3/jw01439001001_03102_00002_nrca3_cal.fits not yet available.
NOT READY: asn member /ifs/archive/ops/jwst/info/owlmgr/paths/sdp/asn_creation/cal/level3/jw01439001001_03102_00005_nrca3_cal.fits not yet available.
NOT READY: asn member /ifs/archive/ops/jwst/info/owlmgr/paths/sdp/asn_creation/cal/level3/jw01439001001_03102_00009_nrca3_cal.fits not yet available.
NOT READY: asn member /ifs/archive/ops/jwst/info/owlmgr/paths/sdp/asn_creation/cal/level3/jw01439001001_03102_00003_nrca3_cal.fits not yet available.
NOT READY: asn member /ifs/archive/ops/jwst/info/owlmgr/paths/sdp/asn_creation/cal/level3/jw01439001001_03102_00001_nrca3_cal.fits not yet available.
NOT READY: asn member /ifs/archive/ops/jwst/info/owlmgr/paths/sdp/asn_creation/cal/level3/jw01439001001_03102_00007_nrca3_cal.fits not yet available.
    READY  : asn member /ifs/archive/ops/jwst/info/owlmgr/paths/sdp/asn_creation/cal/level3/jw01439001001_02102_00001_nrca3_cal.fits
STILL WAITING: incomplete collection of product jw01439-o001_t002_nircam_clear-f210m-sub8fp1a-nrca3_{suffix}-02 ```
Pool and asn files are attached.

**Comment by stscijgbot-jp** [comment]
_Comment by [Howard Bushouse](https://jira.stsci.edu/secure/ViewProfile.jspa?name=bushouse) on [JIRA](https://jira.stsci.edu/browse/JP-2703?focusedCommentId=599890#comment-599890):_

Is this duplicated by (or a duplicate of) JP-2659?

**Comment by stscijgbot-jp** [comment]
_Comment by [Hien Tran](https://jira.stsci.edu/secure/ViewProfile.jspa?name=htran) on [JIRA](https://jira.stsci.edu/browse/JP-2703?focusedCommentId=600287#comment-600287):_

yes, this appears to be a duplicate of jp-2659. i'll mark it as such, but not remove it so there's an example of inflight data.

**Comment by stscijgbot-jp** [comment]
_Comment by [Howard Bushouse](https://jira.stsci.edu/secure/ViewProfile.jspa?name=bushouse) on [JIRA](https://jira.stsci.edu/browse/JP-2703?focusedCommentId=615950#comment-615950):_

Fixed by #7339
